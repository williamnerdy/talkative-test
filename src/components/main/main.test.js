import React from 'react';
import { shallow } from 'enzyme';
import Main from './main';

describe('<Main />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
