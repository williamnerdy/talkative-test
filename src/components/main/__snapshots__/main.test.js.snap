// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Main />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <Row
        reverse={true}
      >
        <Col
          size={12}
        >
          <Card>
            <Title
              color="default"
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
            </Title>
          </Card>
        </Col>
        <Col
          size={12}
        >
          <p
            style={
              Object {
                "padding": "30px",
              }
            }
          />
        </Col>
      </Row>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <Col
            size={12}
          >
            <Card>
              <Title
                color="default"
                style={
                  Object {
                    "textAlign": "center",
                  }
                }
              >
                Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
              </Title>
            </Card>
          </Col>,
          <Col
            size={12}
          >
            <p
              style={
                Object {
                  "padding": "30px",
                }
              }
            />
          </Col>,
        ],
        "reverse": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Card>
              <Title
                color="default"
                style={
                  Object {
                    "textAlign": "center",
                  }
                }
              >
                Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
              </Title>
            </Card>,
            "size": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Title
                color="default"
                style={
                  Object {
                    "textAlign": "center",
                  }
                }
              >
                Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
              </Title>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.",
                "color": "default",
                "style": Object {
                  "textAlign": "center",
                },
              },
              "ref": null,
              "rendered": "Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.",
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <p
              style={
                Object {
                  "padding": "30px",
                }
              }
            />,
            "size": 12,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "style": Object {
                "padding": "30px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "p",
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Row
          reverse={true}
        >
          <Col
            size={12}
          >
            <Card>
              <Title
                color="default"
                style={
                  Object {
                    "textAlign": "center",
                  }
                }
              >
                Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
              </Title>
            </Card>
          </Col>
          <Col
            size={12}
          >
            <p
              style={
                Object {
                  "padding": "30px",
                }
              }
            />
          </Col>
        </Row>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <Col
              size={12}
            >
              <Card>
                <Title
                  color="default"
                  style={
                    Object {
                      "textAlign": "center",
                    }
                  }
                >
                  Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
                </Title>
              </Card>
            </Col>,
            <Col
              size={12}
            >
              <p
                style={
                  Object {
                    "padding": "30px",
                  }
                }
              />
            </Col>,
          ],
          "reverse": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Card>
                <Title
                  color="default"
                  style={
                    Object {
                      "textAlign": "center",
                    }
                  }
                >
                  Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
                </Title>
              </Card>,
              "size": 12,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <Title
                  color="default"
                  style={
                    Object {
                      "textAlign": "center",
                    }
                  }
                >
                  Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.
                </Title>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.",
                  "color": "default",
                  "style": Object {
                    "textAlign": "center",
                  },
                },
                "ref": null,
                "rendered": "Você foi convidado a participar da mais inovadora ferramenta de mídia para micro-influenciadores nas redes sociais.",
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <p
                style={
                  Object {
                    "padding": "30px",
                  }
                }
              />,
              "size": 12,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "style": Object {
                  "padding": "30px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
